FastAutoAugment/augmentations.py

```diff
@@ -189,8 +190,28 @@ def get_augment(name):
     return augment_dict[name]
 
 
-def apply_augment(img, name, level):
+def get_magnitude(name, hardness_score, low, high):
+    min_aug = 1 - hardness_score ** (C.get()['hardness']['f_power'])  
+    max_aug = 1 - hardness_score ** (C.get()['hardness']['g_power'])
+    magnitude = random.uniform(min_aug, max_aug)
+    if name in ["Rotate", "ShearX", "ShearY", "TranslateX", "TranslateY", "TranslateXAbs", "TranslateYAbs", "Cutout", "CutoutAbs"]:
+        low = 0.0
+        magnitude = magnitude * (high - low) + low
+    elif name in ["Posterize", "Posterize2", "Solarize"]:
+        magnitude = high - magnitude * (high - low)
+    elif name in ["Contrast", "Sharpness", "Brightness", "Color"]:
+        low = 0.0
+        high = 0.9
+        magnitude = magnitude * (high - low) + low
+        magnitude = round(1 + magnitude * random.choice([-1, 1]), 5)
+        
```

Comment:
please add description of these different edge cases as I had asked for earlier.
Also, don't hard code 0.9. 0.0 is okay. 0.9 is not. You should take this value from the low-high. For this transforms, put an assertion that low and high should be symmetric - so, either 1.9, 0.1 or 0.3, 1.7 - and then take the high value - 1.

Reply:
done.